src/plugins/integrations/MySQL2Integration.ts

```diff
@@ -56,6 +57,10 @@ class MySQL2Integration implements Integration {
           [DBTags.DB_HOST]: this.config.host,
           [DBTags.DB_PORT]: this.config.port,
           [DBTags.DB_TYPE]: DBTypes.MYSQL,
+          [SpanTags.TOPOLOGY_VERTEX]: true,
+          [SpanTags.TRIGGER_DOMAIN_NAME]: LAMBDA_APPLICATION_DOMAIN_NAME,
+          [SpanTags.TRIGGER_CLASS_NAME]: LAMBDA_APPLICATION_CLASS_NAME,
+          [SpanTags.TRIGGER_OPERATION_NAMES]: [this.config.host],
```

Comment:
this must be `config.database` to be consistent with the span name.

Reply:
Done